src/boot/bootctl.c

```diff
@@ -144,6 +144,7 @@ static int help(int argc, char *argv[], void *userdata) {
                "  set-timeout SECONDS Set the menu timeout\n"
                "  set-timeout-oneshot SECONDS\n"
                "                      Set the menu timeout for the next boot only\n"
+               "  purge-entry ID      Remove specified entry and files referenced by it\n"
```

Comment:
maybe call this "unlink"? it's kinda what it does after all: unlink the entry from the backing files (only removing them when link count drops to zero).

I mean "purge" sounds like a comprehensive, strong and tough operation. But this one is a pretty friendly form of removal: it just drops an entry and its resources once it made sure they aren't used.